product_design/structuring_epics_stories_bugs_and_tasks.md

```diff
@@ -0,0 +1,142 @@
+# Structuring Epics, Stories, Bugs and Tasks
+* Contents
+{:toc}
+
+This page summarizes what an Epic, User Story, Bug, and Task is and documents our best practices for how they should be structured. Not every project will follow these practices. We’ll rely on our team norming sessions to adapt ticket writing practices so they best support the team in the context of their project.
+
+Guidance on our approach to [writing Epics and User Stories](../product_design/how_we_write_user_stories.md) can be found here.
+
+![Initiatives inform Epics which then inform our stories, tasks, and bugs](/assets/images/structuring_tickets/structure-overview.png)
+
+<br>
+
+---
+## Initiatives
+Initiatives are a collection of epics that drive towards a common goal (for example: “expand in to Australia” or “launch a new product that decreases costs by 7% this year”). An initiative is broader than a single epic, and may span multiple teams and multiple quarters.
+
+## Epics
+Epics convey the overall big picture priorities. They contain a group of stories (on a particular theme) that are too big to be described by a single user story. Epics are written concisely and in a way that is simple to understand.
```

Comment:
"They contain a group of stories (on a particular theme) that are too big to be described by a single user story. Epics are written concisely and in a way that is simple to understand."

OK it looks like we mention it here.  (re: Epics = group of stories)